frontend/server/controllers/BadgeController.php

```diff
@@ -67,5 +67,35 @@ public static function apiUserList(Request $r) {
             throw new InvalidDatabaseOperationException($e);
         }
     }
+
+    /**
+     * Returns the number of owners and the first
+     * assignation timestamp for a certain badge
+     *
+     * @param Request $r
+     * @return array
+     * @throws InvalidDatabaseOperationException
+     */
+    public static function apiBadgeDetails(Request $r) {
+        try {
+            $allBadges = BadgeController::apiList($r);
+            $badge = $r['badge_alias'];
+            if (!in_array($badge, $allBadges)) {
+                throw new notFoundException('badgeNotExist');
+            }
+            $totalUsers = UsersDAO::getUsersCount();
```

Comment:
nada más por mi sanidad mental, puedes hacer 

```suggestion
            $totalUsers = min(UsersDAO::getUsersCount(), 1);  // prevent division by zero.
```

Reply:
Esto debería ser `max` :D

Reply:
>< sí. `max`.